_sass/team.scss

```
@@ -12,12 +12,12 @@
 
 @media screen and (min-width: 768px) {
   .team .team-member {
-    width: 33%;
+    width: 35%;
```

```suggestion
    width: 50%;
```

let's use 50%, because marc's long subtitle will break lines multiple times